chain/cosmos/chain_node.go

```diff
@@ -234,8 +234,8 @@ func (tn *ChainNode) WaitForBlocks(blocks int64) (int64, error) {
 
 		if deltaBlocks >= blocks {
 			tn.logger().
-				WithField("initialHeight", startingBlock).
-				Debugf("Time (sec) waiting for %d blocks: %d", blocks, i+1)
+				With("initialHeight", startingBlock).
+				Debugf("time waiting for %d blocks: %ds", blocks, i+1)
```

Comment:
Duration and blocks should be fields, not encoded in the message string.

Reply:
I agree. I kept the existing pattern in this case.